tests/Unit/Collection/ItemPriceCollectionTest.php

```diff
@@ -232,7 +235,7 @@ public function testResetDiscounts()
     public function totalProvider()
     {
         // Items with discounts and tax
-        $tax_price = new TaxPrice(10, 'exclusive');
+        $tax_price = new TaxPrice(10, TaxPrice::EXCLUSIVE);
```

Comment:
It would probably make sense to also update the TaxPriceTest class to use these constants too since you're doing it here.